mlos_bench/mlos_bench/tests/optimizers/toy_optimization_loop_test.py

```diff
@@ -93,10 +93,10 @@ def test_smac_optimization_loop(mock_env_no_noise: MockEnv,
     Toy optimization loop with mock environment and SMAC optimizer.
     """
     (score, tunables) = _optimize(mock_env_no_noise, smac_opt)
-    assert score == pytest.approx(75.0, 0.01)
+    assert score == pytest.approx(67.6, 0.01)
```

Comment:
These changed because previously the `max_trial` was still set to 100 inside SMAC due to lack of config propagation.  This changes the way the initial random sampling proceeds.

Reply:
But now running into an issue with them not being deterministic across python versions ...